src/styles/components/classroom-manager.styl

```diff
@@ -0,0 +1,20 @@
+.classroom-manager
+  &__table
+    background-color: white
+    border: solid 2px #DEDEDE
+
+  .table
+    &__caption
+      bottom: -2.25em
+      font-weight: bold
+      position: relative
+      text-align: left
+
+    &__row-header
+      background-color: GREY_5
+      border-top: solid 2px #DEDEDE !important
```

Comment:
Ahhah, confusing, yes, but no worries - we can always come back and do some renaming & general refactoring after launch. 👍  At this moment, important thing is that everything works nicely.